lms/static/js/views/fields.js

```diff
@@ -74,14 +104,14 @@
             },
 
             title: function (text) {
-                return this.$('.u-field-title').html(text);
+                return HtmlUtils.setHtml(this.$('.u-field-title'), text);
             },
 
             getMessage: function(message_status) {
                 if ((message_status + 'Message') in this) {
                     return this[message_status + 'Message'].call(this);
                 } else if (this.showMessages) {
-                    return this.indicators[message_status] + this.messages[message_status];
+                    return HtmlUtils.joinHtml(this.indicators[message_status], this.messages[message_status]);
```

Comment:
I'm not clear if this is trying to fix a defect, or other.  I need more info on this regarding what is HTML and what is not here.